### ISSUE TITLE
Move "mongo.shell.path" to workspace settings

### ISSUE THREAD

**Issue by alexshin** [Microsoft/vscode-cosmosdb#311]
Guys you do a really good work) I'd like to suggest you making possible moving an option with "mongo.shell.path" to workspace settings (for today it works only from user settings).

E.g. for my case i prefer working with a local MongoDB inside docker container. Every project has docker-compose.yml files to build environment with mongo, redis etc. Working this way makes me to change user settings every time when switching between projects. Option with "mongo.shell.path" seems like:

```json
{
    "mongo.shell.path": "docker exec -it mongodb mongo"
}
```

If the option is available from workspace settings, it could be more convenient, cause you can store different settings for different projects.

**Comment by EricJizbaMSFT** [comment]
@alexshin could you expand on "for today it works only from user settings"? All user settings should be available in workspace settings as well

**Comment by EricJizbaMSFT** [comment]
I [ran into this](https://github.com/Microsoft/vscode-azurefunctions/pull/210) in the functions extension. I'm guessing the fix is to change the scope to 'resource' instead of 'workspace.

**Comment by StephenWeatherford** [comment]
FYI, we released 0.6.0 with this fix a couple of days ago.  Please let us know if you run into troubles.
